shopelectro/tests/tests_selenium.py

```diff
@@ -605,38 +626,38 @@ def test_login(self):
         admin_title = self.browser.find_element_by_id('site-name')
         self.assertIn(self.title_text, admin_title.text)
 
-    def test_admin_product(self):
-        """
-        Admin products page has icon links for Edit.
-        And it should has Search field.
-        """
+    def test_admin_index(self):
+        """At admin index page, must be app list with 3 items"""
```

Comment:
`There are should be app list with 3 items on admin index page`
I think it's better.